app/javascript/application.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import { BrowserRouter } from 'react-router-dom';
+import { Provider } from 'react-redux';
+import store from './redux/store';
+import App from './components/app';
+
+const root = ReactDOM.createRoot(document.getElementById('root'));
```

Comment:
- [ ] This link leads to a blank page:

![image](https://github.com/MenendezON/book-an-appointment/assets/57421823/9eb1b772-4a48-45b0-84b2-082a3bce9025)

I am inferring this is a frontend bug, since the server does not move when trying to visit that link.

Kindly, make sure this link can be used to reserve bikes, as intended. 👍